spec/lib/contextual_logger/mixins/active_support_tagged_logging_spec.rb

```diff
@@ -9,9 +9,10 @@
 describe 'ContextualLogger::Overrides::ActiveSupport::TaggedLogging::Formatter' do
   before do
     Time.now_override = Time.now
-    @logger = ContextualLogger.new(Logger.new('/dev/null'))
+    @logger = Logger.new('/dev/null')
     @logger.formatter = ->(_, _, _, msg_with_context) { "#{msg_with_context.to_json}\n" }
     @logger = ActiveSupport::TaggedLogging.new(@logger)
+    @logger = ContextualLogger.new(@logger)
```

Comment:
Yuck, I hadn't noticed how this overwrites `@logger`! I will fix that on my branch.